front-end-react/src/App.js

```diff
@@ -0,0 +1,15 @@
+import './App.css';
```

Comment:
Did we want to make these ts files for typescript?  I know we discussed it, just want to be sure if we still want to do that.

Reply:
yes good catch.